.github/workflows/ci.yaml

```diff
@@ -22,3 +22,21 @@ jobs:
       run: make generate manifests && git diff --exit-code
     - name: Test
       run: make test
+    - name: Get version from tags
+      id: version
+      run: echo ::set-output name=tag::${GITHUB_REF/refs\/tags\//}
+    - name: Publish
+      if: startsWith(github.ref, 'refs/tags/')
+      env:
+        KO_DOCKER_REPO: ghcr.io/${{ github.repository_owner }}
+        AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        REPOSITORY_OWNER: ${{ github.repository_owner }}
+        VERSION: ${{ steps.version.outputs.tag }}
+      run: |
+        echo "${AUTH_TOKEN}" | go run -modfile hack/ko/go.mod github.com/google/ko login ghcr.io --username ${REPOSITORY_OWNER} --password-stdin
+        make publish
+    - name: Update the release with Kubernetes resources
+      uses: softprops/action-gh-release@v0.1.12
```

Comment:
Let's use the official actions to create a release and attach a file to the release.

Reply:
This official once looks archived and recommend this one: https://github.com/actions/upload-release-asset

Reply:
today I learned...